src/sections/Workshop-grid/index.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import { MDXRenderer } from "gatsby-plugin-mdx";
+import { Container, Row, Col } from "../../reusecore/Layout";
+import WorkshopCard from "../../components/Workshop-Card";
+
+import { WorkshopPageWrapper } from "./WorkshopsGrid.style";
+
+// Social Icons
+import Slack from "../../images/socialIcons/slack_color.png"
```

Comment:
Instead of using these icons. Use icons from `react-icons` .https://react-icons.github.io/react-icons/ .

Reply:
Yeah, I could but It won't have any color. Is that ok?

Reply:
hmm..it will difficult to color the slack logo. Leave as it is.